gradle/libs.versions.toml

```diff
@@ -92,6 +94,8 @@ aliyun-sdk-oss = { module = "com.aliyun.oss:aliyun-sdk-oss", version.ref = "aliy
 analyticsaccelerator-s3 = { module = "software.amazon.s3.analyticsaccelerator:analyticsaccelerator-s3", version.ref = "analyticsaccelerator" }
 antlr-antlr4 = { module = "org.antlr:antlr4", version.ref = "antlr" }
 antlr-runtime = { module = "org.antlr:antlr4-runtime", version.ref = "antlr" }
+antlr-antlr413 = { module = "org.antlr:antlr4", version.ref = "antlr413" }
+antlr-runtime413 = { module = "org.antlr:antlr4-runtime", version.ref = "antlr413" }
```

Comment:
This feels a little odd to me, Do we really need a second antlr version? Is there incompatibility with the runtime?

Reply:
better to align with Spark, otherwise at least a warning message printed from the ANTLR-generated code at runtime.

Reply:
We still need the original `antlr = "4.9.3"` to build Spark 3.4 and 3.5.